src/index.ts

```diff
@@ -0,0 +1,9 @@
+import { Registry } from './registry';
+
+export * from './registry';
+export * from './collector';
+export * from './counter';
+export * from './gauge';
+export * from './histogram';
```

Comment:
Whats the reason for exporting everything now? The old api was just the main function creating an instance of `Registry`.

Reply:
Because I nee to import types in my code. Do you know another way?

Reply:
I was under the impression that thats what the `lib/*.d.ts` files were for? In the final module the type definition files are separate from the JS. It seems less than ideal to have to export everything just for the types when the type definitions are clearly separate. 
@dimitropoulos Do you know of any better ways for this to work? I cannot get an alternative working locally but then typescript is not my area of expertise.

Reply:
I've fixed that

Reply:
How are you consuming the types now? I tried `import prom, { Counter } from 'promjs';` to no avail?

Reply:
```typescript
import prom from 'promjs';
import { CounterType, GaugeType, RegistryType } from 'promjs';
```

Reply:
And `new CounterType()` gives an error.

Reply:
Yep nice. I didn't realise the `Type` suffix on the imports. Is that default behaviour with typescript?

Reply:
I export type aliases here 
https://github.com/weaveworks/promjs/pull/17/files#diff-41df78d86ed4de80aae9b0b9560feb2bR8
so it works only for TS

Reply:
Gotcha. Thanks for answering all my questions 👍